devicemodel/hw/pci/virtio/virtio_net.c

```diff
@@ -940,8 +940,11 @@ virtio_net_init(struct vmctx *ctx, struct pci_vdev *dev, char *opts)
 		return -1;
 	}
 
-	vtopts = tmp = strdup(opts);
```

Comment:
Does the vtops/tmp need  to be initialized? Otherwise the uninitialized value will cause that  it has some issues in free(vtopts)